content/theme/force/ways-of-the-force.md

```diff
@@ -0,0 +1,43 @@
+a viable approach. To those with the appropriate sensitivity, it can be applied in a seemingly infinite variety of different ways. Individuals who are strong in the Force are 
+able to enhance their own abilities, understand their environment on a deeper or supernatural level, or even use it to impose their will upon their physical surroundings.
+
+#Learning About the Force
+However, this ability is quite rare. Further, though an individual may manifest sensitivity to the Force at any time, such a talent most be developed, rather than taught as a 
+skill. For most species, it typically appears at a young age. Although even untrained Force use can sometimes grant insights or physical capabilities far beyond those considered 
+normal, to effectively wield the Force in combat or in a more controlled manner requires a high level of training and discipline. 
+
+Therefore, communities of Force users are relatively small and isolated from the rest of the galaxy. The Jedi Order seeks out young and impressionable Force-sensitive children 
+for instruction at one of their many temples throughout the galaxy, although they are usually reluctant to begin training anyone who is deemed too old. Even so, not all children 
+who are sought by the Order to begin training are ever able to finish it; only a select few ever reach the rank of Knight. 
+
+#The Powers of the Force
+The power that the Force grants to a wielder may vary widely, but the types of abilities imparted are traditionally broken into three broad categories.
+
+##Powers of the Body
+Force users can harness the Living Force within themselves to enhance their own physical prowess beyond natural levels. Abilities of this type allow the Jedi and Sith powers 
+such as astounding agility and strength, healing from injuries that would have been meant certain death for a normal member of their species, or hibernating through long periods of 
+deprivation from basic physical needs.
+
+##Powers of Manipulation
+These powers allow Force users to bend the surrounding physical environment to their own will. This allows for supernatural feats such as telekinesis, energy redirection and
+projection. Such abilities can also involve the subtle manipulation of the minds of other beings. Those enthralled by raw power have always been particularly attracted to this 
+type of Force manipulation.
+
+##Powers of the Mind
+Powers of the mind include those that allow an awareness of the greater galaxy around the Force user. To witness events of the past, present, and future through visions achieved
+via meditation or other means is among the capabilities imparted by attunement to the Cosmic Force. At their fullest potential, these abilities may even grant the wielder the 
+power to orchestrate or influence events according to his or her own desires. 
+
+##Lightsabers and the Force
```

Comment:
I think a greater discussion about Lightsabers and whether or not they can be wielded by non Force users is warranted, but this is fine for now.